File: src/main/java/top/zbeboy/isy/web/vo/data/politics/PoliticsVo.java
package top.zbeboy.isy.web.vo.data.politics;

import lombok.Data;

import javax.validation.constraints.NotNull;
import javax.validation.constraints.Size;

/**
 * Created by lenovo on 2016-11-03.
 */
@Data
public class PoliticsVo {
    private Integer politicalLandscapeId;
    @NotNull
    @Size(max = 30)
    private String  politicalLandscapeName;
}

File: src/main/java/top/zbeboy/isy/web/vo/data/politics/PoliticsVo.kt
package top.zbeboy.isy.web.vo.data.politics

import javax.validation.constraints.NotNull
import javax.validation.constraints.Size

/**
 * Created by zbeboy 2017-12-12 .
 **/
open class PoliticsVo {
    var politicalLandscapeId: Int? = null
    @NotNull
    @Size(max = 30)
    var politicalLandscapeName: String? = null
}